### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,203 +1,5 @@
...
-build/
-bld/
-[Bb]in/
-[Oo]bj/
-
-# Visual Studo 2015 cache/options directory
 .vs/
-
-# JetBrains project files
-.idea/
```
Comment (0xced) [inline .gitignore:29]: If .vs/ stays then .idea/ should stay too.

File: src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj [hunk 2]
```diff
@@ -24,12 +23,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Using Remove="System.Net.Http" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
-    <PackageReference Include="PolySharp" Version="1.13.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
```
Comment (0xced) [inline src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj:26]: If global.json has been updated to the dotnet 8 SDK then the SourceLink package is not required anymore.

File: .gitignore [hunk 3]
```diff
@@ -1,203 +1,5 @@
...
-[Rr]eleases/
-x64/
-x86/
-build/
-bld/
-[Bb]in/
-[Oo]bj/
-
-# Visual Studo 2015 cache/options directory
 .vs/
```
Comment (sungam3r) [inline .gitignore:1]:
```suggestion
.vs/
. idea/
```

File: .gitignore [hunk 4]
```diff
@@ -1,203 +1,6 @@
...
-
-# Visual Studio 6 build log
-*.plg
-
-# Visual Studio 6 workspace options file
-*.opt
-
-project.lock.json
-
+. idea/
```
Comment (sungam3r) [inline .gitignore:2]:
```suggestion
.idea/
```

File: src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj (hunk 2 above)
Comment (sungam3r) [reply src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj:26]: Only package or some other properties can be removed as well?

Comment (sungam3r) [reply src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj]: I removed it.

**Comment by sungam3r** [comment]
@nblumhardt I have resolved conflicts.

**Pull request merged**